### ISSUE TITLE
:robot: Automatically generate JSON with latest versions

### ISSUE THREAD

**Issue by sdwilsh** [kairos-io/kairos#1304]
**Is your feature request related to a problem? Please describe.**
The Kairos version contains the version of Kairos (like 1.6.1) as well as the version of k3s (like 1.26.1, so the full version was `v1.6.1-k3sv1.26.1-k3s1`).  Unfortunately, at least as far as I can tell, this makes it impossible for Renovate to help someone keep appraised of version updates like they might for other libraries.  For 2.0, the equivalent version was `v2.0.0-k3sv1.26.3-k3s1`.  Note that both the versions of k3s and Karios changed.  While Renovate could track the latest version of both of those, that runs the risk of creating a version that doesn't actually exist.

**Describe the solution you'd like**
I think the easiest way to handle this might be some kind of alias system such that `v2.0.0-k3sv1.26` maps to the specific point release of k3s that actually ships with.  Anyone who wants to manage the Kairos version with a specific version of Kubernetes can at least get the notifications about Kairos being up-to-date.

**Describe alternatives you've considered**
I've tried a few options with Renovate and their Regular Expression capabilities, but they have not panned out.  I'm not sure of any options on the Kairos side of things other than what I've suggested.

**Additional context**
I use an [ansible play to run an upgrade on my cluster](https://github.com/sdwilsh/ansible-playbooks/blob/main/plays/kairos/upgrade.yml), and I have Renovate configured on the repository to help me stay up-to-date on libraries I depend on.

**Comment by mudler** [comment]
Hey @sdwilsh, good thoughts in here.

Having only part of the version could be really problematic - then you can't anymore know what's the minor version used, which is ok for most cases I guess - but we should probably annotate that as part of image labels, or similar. 

I'm fine with dropping part of the versions, however, I'm thinking that maybe we should just generate a json when we release, with all the versions informations in there and publish it somewhere (ideally, our website? I guess it's rather easy to automatize that with a PR). So there is no more guessing around versions

**Comment by sdwilsh** [comment]
Yeah, I agree that the information contained in the version numbers is useful, so it would need a new home.

**Comment by Itxaka** [comment]
I dont think this should be kept open, especially with the new Factory in which we encourage people to maintain their own version and such, the artifacts produced and the k3s/k0s version can be whatever you choose, so Im not sure it applies anymore

@sdwilsh should we close this?
